rust/candid/Cargo.toml

```
@@ -20,10 +20,11 @@ lalrpop = "0.19.0"
 
 [dependencies]
 byteorder = "1.4.3"
-candid_derive = { path = "../candid_derive", version = "=0.4.5" }
+candid_derive = { path = "../candid_derive", version = "0.5" }
```

```suggestion
candid_derive = { path = "../candid_derive", version = "=0.5.0" }
```
Always need to pin the derive version.